bmwqemu.pm

```diff
@@ -260,15 +260,16 @@ sub current_test {
 sub update_line_number {
     return unless current_test;
     return unless current_test->{script};
-    my $out    = "";
+    my @out;
     my $ending = quotemeta(current_test->{script});
-    for my $i (1 .. 10) {
+    my $casedir = $vars{CASEDIR} // '';
+    for (my $i = 10; $i > 0; $i--) {
         my ($package, $filename, $line, $subroutine, $hasargs, $wantarray, $evaltext, $is_require, $hints, $bitmask, $hinthash) = caller($i);
```

Comment:
```suggestion
        my ($package, $filename, $line, $subroutine) = caller($i);
```

Reply:
I didn't change this line and see no reason to do so :)